3.11/http/indexes.md

```diff
@@ -39,7 +39,11 @@ The edge index is non-unique.
 ### Persistent Index
 
 A persistent index is a sorted index that can be used for finding individual documents or ranges of documents.
-In contrast to the other indexes, the contents of a persistent index are stored on disk and thus do not need to be rebuilt in memory from the documents when the collection is loaded.
+
+### Inverted Index
+
+An inverted index can accelerate a broad range of queries, but it is not updated
+immediately like other indexes.
```

Comment:
I don't think "updated immediately" is a good explanation, maybe something like, but it results can be outdated, because its eventual consistent?

Reply:
updated with a short delay, which means values returned from an inverted index can be stale?

Documents may also temporarily disappear from the index (because of RocksDB snapshots as per Andrey), which is a not a typical property of eventual consistency...

Reply:
I don't think "short delay" is good, because what does mean "short" in cluster?

In general it's not critical or something, just some thoughts